packages/react-composites/src/composites/MeetingComposite/Strings.tsx

```diff
@@ -29,6 +29,10 @@ export interface MeetingCompositeStrings {
    * Side pane Chat screen title.
    */
   chatPaneTitle: string;
+  /**
+   * New Message label
+   */
+  newMessage: string;
```

Comment:
nit: newMessage conflicts with the new messages string in the message thread, consider `chatButtonNewMessageLabel`

Reply:
Update the doc comment also to say where exactly this string is used (so Contoso knows what this is used for)